docker/customization/StudyViewInitCharts.js

```diff
@@ -0,0 +1,1237 @@
+/* 
```

Comment:
No. Part of internal testing for the IPM cBioPortal that we were changing that file.
I will remove it. Thanks